web/src/routes/LicensePage.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { License, parseLicenses } from 'api-client/src/utils/licences'
+
+import Caption from '../components/Caption'
+import List from '../components/List'
+import ListItem from '../components/ListItem'
+import { reportError } from '../utils/sentry'
+
+const LicenseContainer = styled.div`
+  padding: 5px 15px;
+  line-height: 120%;
+`
+
+type PropsType = {
+  name: string
+  version: string | undefined
+  license: string
+  onPress: string
+}
+
+const LicenseItem = (props: PropsType): ReactElement => {
+  const { name, version, license, onPress } = props
```

Comment:
Optional: If you destructure `props` directly you could remove brackets/return statement.

Reply:
done